src/Message.h

```diff
@@ -54,6 +54,19 @@ class MessageHandler {
 #undef MUMBLE_MH_MSG
 };
 
+/// UDPMessageTypeIsValidVoicePacket checks whether the given
+/// UDPMessageType is a valid voice packet.
+inline bool UDPMessageTypeIsValidVoicePacket(MessageHandler::UDPMessageType umt) {
+	switch (umt) {
```

Comment:
We should probably add MessageHandler::UDPPing and return false for it.

If we add default, we won't get warnings for missing entries in the future...